src/dashboard/PointOfCareDashboard.jsx

```diff
@@ -226,7 +226,7 @@ export default class PointOfCareDashboard extends Component {
                     forceRefresh={this.props.forceRefresh}
                     appState={this.props.appState}
                     highlightedSearchSuggestion={this.props.highlightedSearchSuggestion}
-                    isNoteViewerEditable={false}
+                    isNoteViewerEditable={true}
```

Comment:
[JA3] Was there a reason for changing this?

Reply:
I don't believe so and I also don't remember why I changed it, but I changed it back so [JA3] resolved!